lib/boot-record.coffee

```diff
@@ -12,6 +13,7 @@ BOOT_RECORD_SIZE = 512
 # @description It returns a 512 bytes buffer.
 #
 # @param {String} image - image path
+# @param {filedisk.Disk} image - Disk instance
```

Comment:
@jviotti how do I specify that the first parameter can be either a String or a filedisk.Disk ?